DOCS: update for lightspeed app docs update 1.1.0 (#1541)
MultiSafepay/docs: content/payments/integrations/ecommerce-platforms/lightspeed_app/faq/changing-api-key-and-environment.md

To change your [API key](/faq/general/multisafepay-glossary/#api-key) or environment after installation, follow these steps:

> 1. Sign in to the MultiSafepay app for Lightspeed app.

Reviewer: ```suggestion
1. Sign in to the MultiSafepay app for Lightspeed.
```

> 2. Press the hamburger menu and goto **Environment**.

Reviewer: ```suggestion
2. Tap the hamburger menu, and select **Environment**.
```

> 3. In this section you can edit your **API key**, and the **Environment**.

Reviewer: ```suggestion
3. In this section, you can edit your **API key** and the **Environment**.
```
